### PR TITLE
[BUGFIX] PHP v8 compatibility

### PR DESCRIPTION
**Pull request by mabolek**
And a little bit of cleaning.

### CODE REVIEW

**Review by georgringer — APPROVED** [review]

**Comment by georgringer** [comment]
wanna take care of the conflict @mabolek

**Comment by mabolek** [comment]
Should be resolved now.

File: Classes/Controller/NewsBaseController.php [hunk 1]
```diff
@@ -151,10 +150,11 @@ protected function handleNoNewsFoundError($configuration): string
                 $standaloneTemplate = GeneralUtility::makeInstance(StandaloneView::class);
                 $standaloneTemplate->setTemplatePathAndFilename(GeneralUtility::getFileAbsFileName($options[1]));
                 return $standaloneTemplate->render();
-                break;
             default:
                 return '';
         }
+
+        return null;
```
Comment (georgringer) [inline Classes/Controller/NewsBaseController.php:157]: must be '' now

**Review by georgringer — CHANGES_REQUESTED** [review]

**Comment by georgringer** [comment]
has been resolved today with a different commit, sry not using yours

**Pull request closed without merging**